### PR TITLE
SP-283 - Article page

### PR DESCRIPTION
**Pull request by Gatreek92**

### CODE REVIEW

File: src/pages/article.js [hunk 1]
```diff
@@ -0,0 +1,593 @@
...
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </article>
+  )
+}
+
+const ArticleComments = () => {
```
Comment (bouthouri) [inline src/pages/article.js:444]: There will be no comment for the moment the client needs to validate this first.
So we should comment this until we have the green light.

**Review by bouthouri — CHANGES_REQUESTED** [review]
Gatsby is going to generate an article page for every article.
The article page is going to be used as a template so please change it from an actual page to a template.
Inside template folder.

**Review by bouthouri — CHANGES_REQUESTED** [review]
Comment rating related code until we have a client confirmation
Extract the author block into a separate component
Use mocked data in components, not hardcoded data

**Comment by bouthouri** [comment]
Task linked: [SP-283 Article page](https://app.clickup.com/t/2427294/SP-283)

File: src/components/ArticlePreview/ArticlePreview.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  line-height: 1.5;
+  a {
+    color: #4b3e31;
+  }
+`
+const Description = styled.p`
+  color: #756b62;
+  margin-bottom: 0.6875rem;
+`
+const ImageContainer = styled.div`
```
Comment (fakhergh) [inline src/components/ArticlePreview/ArticlePreview.js:43]: @Gatreek92 should have the postfix `Wrapper` for naming convention as Oussama said before.

File: src/components/ArticlePreview/ArticlePreview.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+          <a dangerouslySetInnerHTML={{ __html: title }} href={link}></a>
+        </Title>
+        <Description>{description}</Description>
+        <a href={link}>Mehr</a>
+      </div>
+    </ArticlePreviewContainer>
+  )
+}
+
+const ArticlePreviewContainer = styled.article`
```
Comment (fakhergh) [inline src/components/ArticlePreview/ArticlePreview.js:24]: @Gatreek92 should have the postfix Wrapper for naming convention as Oussama said before.

File: src/components/Posts/Posts.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React from "react"
-import GatsbyImage from "gatsby-image"
 import Masonry from "react-masonry-component"
```
Comment (fakhergh) [inline src/components/Posts/Posts.js:2]: @Gatreek92 can you please add a new line between node modules and local imports?

File: src/templates/ArticleTemplate.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import React from "react"
+import styled from "styled-components"
+import { Layout, ArticleContent, RelatedPosts } from "../components"
```
Comment (fakhergh) [inline src/templates/ArticleTemplate.js:3]: @Gatreek92 newline please between imports.

**Pull request merged**